### PR TITLE
Admin mode indicator. Closes #35

### LINKED ISSUE #35: Visual cue of being logged in as admin
@maxalbert and @jch1g10 suggested that it would be a good idea to provide a stronger visual cue that you are logged in as an administrator in FlowAuth, e.g. by changing the colour of the menu bar.

### PR DESCRIPTION
**Pull request by greenape**
Closes #35
### I have:

- [ ] Formatted any Python files with [black](https://github.com/ambv/black)
- [ ] Brought the branch up to date with master
- [x] Added any relevant Github labels
- [ ] Added tests for any new additions
- [ ] Added or updated any relevant documentation
- [ ] Added an Architectural Decision Record (ADR), if appropriate
- [ ] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate 

### Description

Provides a visual cue of being logged in as a FlowAuth administrator by:
- changing the app bar red
- changing the title to FlowAuth: Admin Mode

when logged in as an administrator.

screenshot 2018-11-07 at 13 44 05

### CODE REVIEW

File: flowauth/frontend/src/Dashboard.js [hunk 1]
```diff
@@ -171,6 +171,7 @@ class Dashboard extends React.Component {
               classes.appBar,
               this.state.open && classes.appBarShift
             )}
+            color={is_admin ? "secondary" : "primary"}
```
Comment (maxalbert) [inline flowauth/frontend/src/Dashboard.js]: I trust this works, but where are the "primary" and "secondary" colours actually set? They don't seem to be mentioned elsewhere in this file, nor in any of the css files?

Comment (greenape) [reply flowauth/frontend/src/Dashboard.js]: They come from the material-ui package.

Comment (maxalbert) [reply flowauth/frontend/src/Dashboard.js]: I see, thanks! Not sure if it's worth adding a comment to this effect or if this is effectively common knowledge. Out of interest, are we using a default palette or defining our own somewhere?

Comment (greenape) [reply flowauth/frontend/src/Dashboard.js]: Default palette, but we'll need to have a branding exercise at some point I think.

**Pull request merged**